ui/src/app/services/error-reporter.service.ts

```diff
@@ -10,6 +10,12 @@ export class ErrorReporterService extends ErrorHandler {
 
   constructor(serverConfigService: ServerConfigService) {
     super();
+    if (window.location.protocol !== 'https:') {
```

Comment:
I believe it will also fail to submit to stackdriver if we are sending from an `http://` host anyway, so I am perfectly happy adding this check regardless of the local piece.